code/game/objects/items/circuitboards/computer_circuitboards.dm

```diff
@@ -387,20 +387,14 @@
 		to_chat(user, "<span class='notice'>You adjust [src]'s routing and receiver spectrum, unlocking special supplies and contraband.</span>")
 
 /obj/item/circuitboard/computer/cargo/express
-	name = "Express Supply Console (Computer Board)"
+	name = "Outpost Comms Console (Computer Board)"
```

Comment:
i don't think a distinction between "cargo console" and "mission console" is really necessary. my intent is for the tabs to be part of a unified outpost UI for non-physical outposts; cleaning up some of the express supply console code would probably make this a bit clearer. i'm probably going to retype it away from "/obj/machinery/computer/cargo/express", too, but merge conflicts are annoying

Reply:
There's still merit in making the mission console a subtype and leaving the original express console unchanged, since i can easily see the original behavior still being desired in a few cases (as, say, a specialty console that's only available to specific cargo ships or factions such as Donk)

Reply:
if we want the original behavior we can readd it at another date tbh